sqlmesh/core/engine_adapter/mssql.py

```diff
@@ -0,0 +1,62 @@
+"""Contains MsSqlEngineAdapter."""
+
+
+from __future__ import annotations
+
+import typing as t
+
+import pandas as pd
+
+from sqlmesh.core.engine_adapter.base import EngineAdapterWithIndexSupport
+from sqlmesh.core.engine_adapter.shared import DataObject, DataObjectType
+
+
+class MsSqlEngineAdapter(EngineAdapterWithIndexSupport):
```

Comment:
[Nit] Ditto wrt name

Reply:
You can test this with `sqlmesh fetchdf "<query>"`.

Reply:
Similarly it doesn't look like MSSQL supports `CREATE OR REPLACE TABLE`, so you might also need `LogicalReplaceQueryMixin`.